index.js

```diff
@@ -444,6 +444,15 @@ export default class OneSignal {
         RNOneSignal.setExternalUserId(externalId, callback);
     }
 
+    static setExternalUserId(externalId, authHashToken, callback) {
```

Comment:
```js
    static setExternalUserId(externalId, authHashToken, callback=function(){}) {
```

instead of 

```js
if (callback === undefined)
            callback = function(){};
```